src/app/components/ScrollablePromo/helpers/fixtureData.js

```diff
@@ -0,0 +1,1217 @@
+                            position: [4, 2, 2, 1, 1, 1],
+                          },
+                        ],
+                        isExternal: false,
+                      },
+                      position: [4, 2, 2, 1, 1],
+                    },
+                  ],
+                },
+                position: [4, 2, 2, 1],
+              },
+            ],
+          },
+          position: [4, 2, 2],
+        },
+      ],
+    },
+    position: [4, 2],
+  },
+  {
+    id: '31553901',
+    type: 'link',
+    model: {
+      locator: 'urn:bbc:content:url-promo:https%3A%2F%2Fwww.bbc.com%2Fmundo',
+      blocks: [
+        {
+          id: 'd7bff4c9',
+          type: 'text',
+          model: {
+            blocks: [
+              {
+                id: '2d16f249',
+                type: 'paragraph',
+                model: {
+                  text: 'Link with Headline',
+                  blocks: [
+                    {
+                      id: '8f2d3f60',
+                      type: 'urlLink',
+                      model: {
+                        text: 'Link with Headline',
+                        locator: 'https://www.bbc.com/mundo',
+                        blocks: [
+                          {
+                            id: '5114f0ff',
+                            type: 'fragment',
+                            model: {
+                              text: 'Link with Headline',
+                              attributes: [],
+                            },
+                            position: [4, 3, 1, 1, 1, 1],
+                          },
+                        ],
+                        isExternal: false,
+                      },
+                      position: [4, 3, 1, 1, 1],
+                    },
+                  ],
+                },
+                position: [4, 3, 1, 1],
+              },
+            ],
+          },
+          position: [4, 3, 1],
+        },
+      ],
+    },
+    position: [4, 3],
+  },
+  {
+    id: '5c979041',
+    type: 'link',
+    model: {
+      locator: 'urn:bbc:content:url-promo:https%3A%2F%2Fwww.bbc.com%2Fbrasil',
+      blocks: [
+        {
+          id: '7ad184aa',
+          type: 'text',
+          model: {
+            blocks: [
+              {
+                id: '57c9fb66',
+                type: 'paragraph',
+                model: {
+                  text: 'https://www.bbc.com/brasil',
+                  blocks: [
+                    {
+                      id: 'a0f0efea',
+                      type: 'urlLink',
+                      model: {
+                        text: 'https://www.bbc.com/brasil',
+                        locator: 'https://www.bbc.com/brasil',
+                        blocks: [
+                          {
+                            id: '883f74a8',
+                            type: 'fragment',
+                            model: {
+                              text: 'https://www.bbc.com/brasil',
```

Comment:
```suggestion
                              text: 'Brasil Homepage',
```

Is this meant to have some text rather than the url, looks like this in storybook:
<img width="818" alt="image" src="https://user-images.githubusercontent.com/9645462/149095345-6fdffb3d-a7ba-42bc-9f68-750ba1bb6c38.png">

Reply:
This was a test given to me in the original testing data. However, I can see how it might be confusing so I will modify this.